fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1236,7 +1236,7 @@ private ExtractedParameterValue processPrimitiveValue(FHIRPathSystemValue system
             DateParmVal p = new DateParmVal();
             TemporalAccessor v = systemValue.asTemporalValue().temporal();
             java.time.Instant inst = DateTimeHandler.generateValue(v);
-            p.setValueDate(DateTimeHandler.generateTimestamp(inst));
```

Comment:
Its interesting to see that we DID have a case where we'd populate the Value and not the Start or End.  Good catch.

Reply:
But actually, I think we should set both the Start *and* End values in this case.  Agree?

Reply:
Yes, updated to set the same end time - as this is actually an Instant (at this point in the code)